src/HDFrestart.cc

```diff
@@ -499,6 +503,7 @@ HDFrestart::HDFrestart(const std::string& filename, const pb::PEenv& pes,
                 << "HDFrestart(): create file " << filename_ << std::endl;
         hid_t access_plist
             = H5Pcreate(H5P_FILE_ACCESS); // property list identifier
+#ifdef MGMOL_USE_HDF5P
```

Comment:
should similar changes to guard use_hdf5p_ be applied in HDFrestart.h as well? There are a few functions in there that use a this runtime check.